packages/tideline/js/plot/quickbolus.js

```diff
@@ -55,7 +55,7 @@ function plotQuickBolus(pool, opts = defaults) {
         .attr({
           'class': 'd3-bolus-group',
           'id': (d) => `bolus_group_${d.id}`,
-          'data-testid': 'd3-bolus-group'
+          'data-testid': (d) => `${d.prescriptor === 'manual' ? 'd3-bolus-group-manual': 'd3-bolus-group' }`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏If possible, reuse the enum value `Prescriptor.Manual` instead of a magic string here